src/api/app/services/workflows/yaml_downloader.rb

```diff
@@ -16,7 +16,7 @@ def call
     def download_yaml_file
       Down.download(download_url, max_size: MAX_FILE_SIZE)
     rescue Down::Error => e
-      raise Token::Errors::NonExistentWorkflowsFile, ".obs/workflows.yml could not be downloaded on the SCM branch #{@scm_payload[:target_branch]}: #{e.message}"
+      raise Token::Errors::NonExistentWorkflowsFile, ".obs/workflows.yml could not be downloaded from the SCM branch #{@scm_payload[:target_branch]}: #{e.message}"
```

Comment:
The specs have to be adapted for this change: https://app.circleci.com/pipelines/github/saraycp/open-build-service/1117/workflows/5e50a438-494e-4f91-840e-b980a8649dc9/jobs/6787

Reply:
@dmarcoux done!